client/src/components/CommonComponents/Tabs/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import useStyles from './style';
+
+export default function DisabledTabs() {
+	const [value, setValue] = React.useState(0);
+
+	const handleChange = (event, newValue) => {
+		setValue(newValue);
+	};
+
+	const classes = useStyles();
+
+	return (
+		<Tabs
```

Comment:
You add a line as text-decoration when we didn`t have text-decoration in Figma

Reply:
only the font family and the size of the font nothing more

Reply:
if you mean the color well its depends on the color on the theme here I used the default material ui theme which colors pink and purple the customize theme on the home page when we merge the colors will be changed auto

Reply:
no, I mean you add a line under clicked tab which we don't have it in Figma!

Reply:
its a Tab we have agreed about it